app/schemas/models/exam.schema.js

```diff
@@ -0,0 +1,62 @@
+const c = require('./../schemas')
+
+const ExamSchema = c.object({
+  title: 'Exam',
+  description: 'CodeCombat exam with limited levels and complete time',
+})
+
+_.extend(ExamSchema.properties, {
+  title: {
+    type: 'string',
+    description: 'Title of the exam',
+  },
+  description: {
+    type: 'string',
+    description: 'Description of the exam',
+  },
+  clan: c.objectId({ links: [{ rel: 'db', href: '/db/clan/{($)}' }] }),
+  startDate: c.stringDate(),
+  endDate: c.stringDate(),
+  duration: {
+    type: 'number',
+    description: 'Duration of the exam in minutes',
+  },
+  problems: {
+    type: 'array',
+    description: 'Problems that are included in the exam',
+    items: {
+      type: 'object',
+      title: 'ExamProblemCourse',
+      description: 'by course, includes instanceId and level slugs',
+      properties: {
+        courseId: {
+          type: 'string',
+          description: 'ID of the course',
+        },
+        instanceId: {
+          type: 'string',
+          description: 'ID of the course instance',
+        },
+        levels: {
```

Comment:
I dont think this works, what if we want to show level 1 from cs1, level 2 from cs2, level 3 from cs1. We should just have an array with each item representing a level.